build/AzurePipelinesTemplates/MUX-InstallWindowsSDK-Steps.yml

```diff
@@ -1,37 +1,10 @@
 parameters:
-  sdkVersion: 18362
+  sdkVersion: 22000
 
 steps:
   - task: powershell@2
     inputs:
       targetType: filePath
       filePath: build\Install-WindowsSdkISO.ps1
       arguments: ${{ parameters.sdkVersion }}
-    displayName: 'Install Windows SDK (${{ parameters.sdkVersion }})'
-
-  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
```

Comment:
I think we should keep all these insider-sdk-specific tasks for the future (commented out since the pipelines are not updated yet?). They just won't be used for now because no pipeline will use UseInsiderSDK for a while.

Reply:
I've added it back, but commented out. We can uncomment if we need.